pkgs/development/tools/rust/cargo-watch/default.nix

```diff
@@ -1,19 +1,19 @@
-{ stdenv, lib, rustPlatform, fetchFromGitHub, CoreServices, Foundation, rust, libiconv }:
+{ stdenv, lib, rustPlatform, fetchFromGitHub, Cocoa, CoreServices, Foundation, rust, libiconv }:
 
 rustPlatform.buildRustPackage rec {
   pname = "cargo-watch";
   version = "8.3.0";
 
   src = fetchFromGitHub {
-    owner = "passcod";
+    owner = "watchexec";
```

Comment:
please update the homepage as well

Reply:
I'm sorry I overlooked this. Follow up in https://github.com/NixOS/nixpkgs/pull/237236.